tdp/core/variables/cluster_variables.py

```diff
@@ -39,7 +39,7 @@ def initialize_cluster_variables(
         Args:
             collections (Collections): instance of collections
             tdp_vars (Union[str, Path]): path to the tdp vars
-            override_folder (Optional[str | Path]): path of tdp vars overrides
+            override_folders (Optional[tuple]): tuple of path(s) of tdp vars overrides
```

Comment:
It's more common to have lists than tuples.
```suggestion
            override_folders (Optional[List[str | Path]]): list of path(s) of tdp vars overrides
```

Reply:
I changed the type hint in this docstring to 'tuple' and the default value to en empty tuple for two reasons :
- unlike lists, tuples are immutable and, the order of the paths here is important
- when using `multiple=True`, the result of Click's parsing always gives a tuple of Paths (not a list) even if there was only one Path provided

Maybe we can change it to accept both `List[str | Path]` and `Tuple[str | Path]` as long as it's an iterable of `str` and/or `Path` objects ?

Reply:
> * unlike lists, tuples are immutable and, the order of the paths here is important

Lists are ordered like tuples, the immutable behavior is not needed in this function.

> Maybe we can change it to accept both `List[str | Path]` and `Tuple[str | Path]` as long as it's an iterable of `str` and/or `Path` objects ?

The function need an iterable of `str` or `Path` yes, you can use `Iterable[str | Path]` if you want.